install_guide_pi.md

```diff
@@ -84,30 +95,66 @@ $ reboot
 
 ### Install ruuvitag-sensor package
 
-In this example we use default installed version of Python 3, which is 3.4.2. Raspbian has also Python 2.7 installed, but it is already 2017, so we will use Python 3. You can check current version with version option. If you want to use Python 2, install also `sudo apt-get install python-dev`. Python developer package is already installed for Python 3
+<!-- seems extraenous, but that's up to you
+In this example we use default installed version of Python 3, which is 3.4.2. 
+Raspbian has also Python 2.7 installed, but it is already 2017, so we will use Python 3. You can check current version with version option. If you want to use Python 2, install also `sudo apt-get install python-dev`. Python developer package is already installed for Python 3
 ```sh
 $ python3 --version
 ```
-
-Ruuvitag-sensor package requires bluez and bluez-hcidump. Bluez is the Linux Bluetooth system and allows a Raspberry Pi to communicate with Bluetooth classic and Bluetooth low energy (LE) devices. Hcidump is a tool which prints data broadcasted by Bluetooth devices to console. Bluez is already installed on Raspbian, so install only bluez-hcidump
+-->
+Ruuvitag-sensor package requires bluez and bluez-hcidump. 
+Bluez is the Linux Bluetooth system and allows a Raspberry Pi to communicate with Bluetooth classic and Bluetooth low energy (LE) devices and is included with Raspbian. 
+Hcidump is a tool which prints data broadcasted by Bluetooth devices to console and needs to be installed.
 ```sh
-$ sudo apt-get install bluez-hcidump
+$ sudo apt-get install bluez-hcidump && echo +++ install successful +++
```

Comment:
Remove && echo +++ install successful +++

Reply:
I prefer to leave echo success since the very verbose apt-get just suddenly exits without a completion message